Kbs.Wpf/Boat/Details/BoatDetailPage.xaml.cs

```diff
@@ -20,16 +22,36 @@ public partial class BoatDetailPage : Page
     private readonly UserRepository _userRepository = new();
     private readonly ReservationRepository _registrationRepository = new();
     private readonly INavigationManager _navigationManager;
+    private const int _daysForDeletion = 30;
     private BoatDetailViewModel ViewModel => (BoatDetailViewModel)DataContext;
     public BoatDetailPage(INavigationManager navigationManager, int boatId)
     {
         _navigationManager = navigationManager;
         InitializeComponent();
         var boat = _boatRepository.GetById(boatId);
+        ViewModel.BoatTypeId = boat.BoatTypeId;
         ViewModel.BoatId = boat.BoatId;
         ViewModel.Name = boat.Name;
         ViewModel.Status = boat.Status.ToDutchString();
         ViewModel.BoatTypeName = _boatTypeRepository.GetById(boat.BoatTypeId).Name;
+        ViewModel.DeleteRequestDate = (boat.DeleteRequestDate.Equals(null))? null : boat.DeleteRequestDate;
```

Comment:
Check is overbodig, komt neer op de waarde zelf (als null, gebruik null, anders de waarde). Kan vervangen worden met een property call:
```suggestion
        ViewModel.DeleteRequestDate = boat.DeleteRequestDate;
```